code/datums/components/heal_react.dm

```diff
@@ -19,7 +19,7 @@
 
 /datum/component/heal_react/boost
 	///multiplicitive boost to incoming healing
-	var/boost_amount = 0.2
+	var/boost_amount = 0.5
```

Comment:
This does effect _all_ incoming healing, I feel the number should stay low-ish and the rate of favor gain should be handled moreso by increasing the gain per point healed to reduce the chance this massively effects a combat healing strat

Reply:
it's been reduced by balance council suggestions